src/modules/notifications/notifications.controller.ts

```diff
@@ -20,6 +20,7 @@ export class NotificationsController {
     try {
       const createdNotification =
         await this.notificationService.createNotification(notificationData);
+      this.logger.log('Notification created successfully.');
```

Comment:
The formatting looks off. Don't we have command to fix it automatically?

Reply:
It's being formatted automatically that way by prettier.